lib/view/home_screen.dart

```diff
@@ -9,10 +10,25 @@ import 'package:gra_terenowa/view/selectTrip_screen.dart';
 import 'package:gra_terenowa/widgets/selectTrip_widget.dart';
 import 'package:gra_terenowa/widgets/cardHero_widget.dart';
 import 'package:gra_terenowa/widgets/tabOutlineIndicator.dart';
+import 'package:map_launcher/map_launcher.dart';
 
-class HomePage extends StatelessWidget {
```

Comment:
Consider navigating to our own map view (when it becomes available), where the church will have its own description. This approach will allow us to avoid navigating out of our app and will save us from running map_launcher.

Reply:
Created an issue for this https://github.com/GrupaMedialnaSalwator/GraTerenowa/issues/33